From fe1d12934d2fa314cca76918cf9a287fbcd71a4e Mon Sep 17 00:00:00 2001
From: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Date: Wed, 3 Nov 2021 10:47:11 +0100
Subject: [PATCH 006/109] MAINTAINERS: rectify entry for INTEL KEEM BAY OCS ECC
 CRYPTO DRIVER

Commit c9f608c38009 ("crypto: keembay-ocs-ecc - Add Keem Bay OCS ECC
Driver") only adds drivers/crypto/keembay/keembay-ocs-ecc.c, but adds a
file entry drivers/crypto/keembay/ocs-ecc-curve-defs.h in MAINTAINERS.

Hence, ./scripts/get_maintainer.pl --self-test=patterns warns:

  warning: no file matches  F:  drivers/crypto/keembay/ocs-ecc-curve-defs.h

Assuming that this header is obsolete and will not be included in the
repository, remove the unneeded file entry from MAINTAINERS.

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Signed-off-by: Daniele Alessandrelli <daniele.alessandrelli@intel.com>
---
 MAINTAINERS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 420eb6d4009e..02621bea3451 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -9552,7 +9552,6 @@ F:	Documentation/devicetree/bindings/crypto/intel,keembay-ocs-ecc.yaml
 F:	drivers/crypto/keembay/Kconfig
 F:	drivers/crypto/keembay/Makefile
 F:	drivers/crypto/keembay/keembay-ocs-ecc.c
-F:	drivers/crypto/keembay/ocs-ecc-curve-defs.h
 
 INTEL KEEM BAY OCS HCU CRYPTO DRIVER
 M:	Daniele Alessandrelli <daniele.alessandrelli@intel.com>
-- 
2.25.1

